pfio/filesystems/hdfs.py

```diff
@@ -82,6 +83,23 @@ def _run_klist(use_keytab=False):
         return None
 
 
+class HdfsFileStat(FileStat):
+    def __init__(self, info):
+        mode = info['permissions']
+        if info['kind'] == 'file':
+            mode |= 0o100000
+        elif info['kind'] == 'directory':
+            mode |= 0o40000
+
+        self.filename = info['path']
+        self.mode = mode
+        self.size = info['size']
+        self.owner = info['owner']
+        self.group = info['group']
+        self.last_modified = info['last_modified']
+        self.last_accessed = info['last_accessed']
+
```

Comment:
Do you think it would be useful to add other fields from the `hdfs.info`, like "replication" and "block_size"